docs/src/pages/premium-themes/onepirate/modules/views/AppFooter.js

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/styles';
+
```

Comment:
It's because of `import { Theme } from '@material-ui/core/styles';`, maybe we should move it to the bottom?

```suggestion
```

x the other cases

Reply:
Let me try that

Reply:
Lint was complaining because of the order of the imports https://app.circleci.com/pipelines/github/mui-org/material-ui/41634/workflows/27c305d8-3e4b-4004-bcef-9d1dff281f42/jobs/248727 I reverted the changes.